src/nwbinspector/checks/ecephys/__init__.py

```diff
@@ -0,0 +1,13 @@
+import warnings
+
+message = (
+    "All submodules below the level 'nwbinspector.checks'  have been deprecated. "
+    "Please retrieve the specific check function(s) you wish to use from the `available_checks` registry "
+    "at the top-level or import directly from 'nwbinspector.checks'."
+)
+
+warnings.warn(message=message, category=DeprecationWarning, stacklevel=2)
+
+# Still keep imports functional with warning for soft deprecation cycle
+# TODO: remove after 9/15/2024
+from .._behavior import *
```

Comment:
```suggestion
from .._ecephys import *
```

I think all of the `__init__.py` files need to be updated, they are all currently importing from the behavior submodule on the L13

Reply:
@stephprince Great catch! Fixed